android/app/build.gradle

```diff
@@ -65,4 +66,9 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation platform('com.google.firebase:firebase-bom:28.1.0')
+    implementation 'com.google.firebase:firebase-analytics-ktx'
```

Comment:
What are these dependencies? Why they are needed?

Reply:
Removed. They were suggest by Firebase docs but are not required at all